locations/spiders/coop_cz.py

```diff
@@ -0,0 +1,47 @@
+import re
+
+from scrapy import Request, Spider
+
+from locations.items import Feature
+
+
+class CoopCZSpider(Spider):
+    name = "coop_cz"
+    item_attributes = {"brand": "COOP", "brand_wikidata": "Q52851660"}
```

Comment:
Is this the same as brand=COOP Jednota brand:wikidata=Q41629254 in NSI?

Reply:
afaik it shouldn't

Reply:
OK, we either need a category applied in the spider, or this brand added to NSI so we get the category from there.